### ISSUE TITLE
Cannot read property 'name' of null

### ISSUE THREAD

**Issue by kalsolio** [getsentry/sentry#9874]
```
            (fn as any) as SentryWrappedFunction,
            {
              mechanism: {
                data: {
                  function: 'addEventListener',
                  handler: ((fn as any) as SentryWrappedFunction).name || '<anonymous>',
                  target,
                },
                handled: true,
                type: 'instrument',
              },
```

**Comment by markstory** [comment]
Which SDK are you using?

**Comment by kalsolio** [comment]
react and using react-component/drawer

**Comment by kamilogorek** [comment]
Patched in `@sentry/browser@4.0.5`
